### PR TITLE
Disable voting on read-only posts

### PR DESCRIPTION
**Pull request by Reeywhaar**
Rel to #230 

PR disables comments voting on read-only posts.

screen shot 2018-12-19 at 10 47 18

@umputun I made it by just disabling arrows, did you mean to completely hide them?

### CODE REVIEW

**Comment by umputun** [comment]
I meant disabling, no need to hide them

File: web/app/components/comment/comment.jsx [hunk 1]
```diff
@@ -335,10 +340,50 @@ export default class Comment extends Component {
...
+      (store.get('user') && store.get('user').id)
+    );
+  }
+
+  /**
+   * returns reason for disabled downvote
+   *
+   * @return {(string|null)}
+   */
+  getDownvoteDisabledReason() {
```
Comment (Guria) [inline web/app/components/comment/comment.jsx]: Is there a reason for two identical methods?

Comment (Reeywhaar) [reply web/app/components/comment/comment.jsx]: Aleksei, thanks for asking! I doubt too, but my intent was that disabling upvote and downvote reasons may differ, mainly for downvote, for example it you want to limit downvote or disable it at all, or want to disable downvoting admins, or mark some controversial comment as non-downvotable, etc..., so I tried to think ahead of time. Maybe there is no reason for this now, and I'm overcomplicating it.

Comment (Reeywhaar) [reply web/app/components/comment/comment.jsx]: Added commit, as an alternative.

**Pull request merged**